pages/states/[state].js

```diff
@@ -21,7 +21,7 @@ function State({ categories, stateData }) {
     const { name, quote } = stateData;
     const imageUrl = "../images/states/" + state + ".png"
     return (
-        <SharedLayout>
+        <SharedLayout title={name}>

```

Comment:
Pass the state name as the page title, so it will say `Alabama - FreeFrom Policy Map`.